### ISSUE TITLE
Service Request by Monica Savage Cayuga County 

### ISSUE THREAD

**Issue by donriepurcell** [dlounsbu/NYHEALing-IPAtracker-support#14]
- [x] e-mail acknowledging that User's Qualtrics Service Request Form was sent
- [x] Problem FULLY RESOLVED and Issue is Closed
- [ ] Post-Service Request Survey was sent to User

Use comments feature (below) to document:

1. A clear and concise description of the solution.

2. A clear and concise description of any alternative solutions or features you've considered.

3. Any additional context, such as screenshots about the problem.

**Comment by donriepurcell** [comment]
From: Monika Salvage <msalvage@cayugacounty.us>
Date: Friday, August 20, 2021 at 1:54 PM
To: Nicholas Bove <nicholas.bove@einsteinmed.org>, Donrie Purcell <donrie.purcell@einsteinmed.org>
Cc: Monika Salvage <msalvage@cayugacounty.us>
Subject: Q about IPA tracker
CAUTION: This email comes from an external source; the attachments and/or links may compromise our secure environment. Do not open or click on suspicious emails. Please click on the “Phish Alert” button on the top right of the Outlook dashboard to report any suspicious emails.
Hello Nick & Donrie:

Here are a few initial questions as I’m perusing the IPA tracker:

1)	Why are milestones not displayed in chronological order by date? It’s hard to follow a sequence that way.
2)	When I enter completion dates unter Measures, they show in a different format, which is confusing. 
3)	I tried to delete some measurements in B2-Peers and got the following error message:
 



Monika

MONIKA SALVAGE
Project Director

HEALing Communities Study
Helping to End Addiction Long-term

Cayuga County Mental Health Center
146 North Street
Auburn, NY 13021
(315) 253-1522

Overdose trends in Cayuga County
Find the latest data

Overcoming opioid use disorder takes more than willpower
Medications can help

Save a life during an opioid overdose emergency
Request a free Narcan kit

Find us on Facebook * Twitter * Instagram * Website

This communication, together with any attachments hereto or links contained herein, is for the sole use of the intended recipient(s) and may contain information that is confidential, privileged, or legally protected, and as such is not a public document. If you are not the intended recipient, you are hereby notified that any review, disclosure, copying, dissemination, distribution or use of this communication is STRICTLY PROHIBITED. If you have received this communication in error, please notify the sender immediately by return e-mail message and delete the original and all copies of the communication, along with any attachments hereto or links herein, from your system.

**Comment by dlounsbu** [comment]
The sorting of Milestones issue is being addressed by Jeff already. 

The other issues, I think, would simply require meeting with Monika to troubleshoot and help her navigate IPAs/Strategies. 

@Donrie, can you set up a time to meet with her about this? 

Best and thanks,
David

**Comment by donriepurcell** [comment]
From: Donrie Purcell <donrie.purcell@einsteinmed.org>
Date: Friday, August 20, 2021 at 3:13 PM
To: Monika Salvage <msalvage@cayugacounty.us>, Nicholas Bove <nicholas.bove@einsteinmed.org>
Cc: Monika Salvage <msalvage@cayugacounty.us>, david lounsbury <david.lounsbury@einsteinmed.org>
Subject: Re: Q about IPA tracker
Hi Monica,

I am available to do a zoom call with you to walk through some of the questions you have. Please let me know when will be a good time and date. Please bear in mind that I am always here to help, including today.

As it relates to your questions, please note the following:

•	In the IPA tracker,  Milestones are displayed in chronological order by the intended start date .
•	The date format in measures are consistent with the date format in milestones, feedback, etc.
•	To delete a smart strategy you have to be cautious. It may be that something  is not as it should be.

Please reach out to me, whenever you’re available. You and I can walk through what you’re trying to accomplish. That way I can understand what you need to have accomplish and I can also help you understand how the system works. 


Regards
Donrie Purcell, PhD, MPH

Every generation has the obligation to free men's minds for a look at new worlds,....to look out from a higher plateau than the last generation.
Ellison S. Onizuka 


CONFIDENTIALITY: This email (including any attachments) may contain confidential, proprietary and/or privileged information. Any duplication, copying, distribution, dissemination, transmission, disclosure or use in any manner of this email (including any attachments) without the authorization of the sender is strictly prohibited. If you receive this email (including any attachments) in error, please notify the sender and delete this email (including any attachments) from your system. Thank you.

**Comment by dlounsbu** [comment]
Also, please keep encouraging users to use our Service Request form!

**Comment by donriepurcell** [comment]
yes  yes 
I meant to

**Comment by dlounsbu** [comment]
@donriepurcell Can we close this issue? Monica's questions were clarified?

**Comment by NBOVE1** [comment]
Jeff: 9/9/21- Someone was trying to enter 6/31/2021 and 11/31/2021 as valid dates and the system wouldn’t save it. I set it such that it would save the record but the date would be a valid date that they’ll probably have to change after the fact.
